components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -50,4 +58,33 @@ export const useAnalyticsTracking = () => {
         window.addEventListener("click", handleButtonOrAnchorTracking, true);
         return () => window.removeEventListener("click", handleButtonOrAnchorTracking, true);
     }, []);
+
+    useEffect(() => {
+        if (loading || !user || !user.additionalData?.profile?.onboardedTimestamp || !isOrbitalLoaded) {
+            return;
+        }
+
+        console.debug("IDing user");
+        orbital("identify", user.id);
+
+        console.log(`Changing theme to ${isDark ? "dark" : "light"}`);
+        orbital("customConfig", {
+            theme: {
+                colorScheme: isDark ? "dark" : "light",
+                colorsLight: {
+                    primary: "#292524",
+                },
+                colorsDark: {
+                    primary: "#fff",
+                },
+            },
+        });
+
+        for (const id of discoveryIds) {
+            console.debug("Triggering modal", id);
+            orbital("trigger", id, { force: true, position: "bottom_left" });
```

Comment:
**Suggestion:** It would be nice if we could "disable" the entire integration by removing all IDs, is this possibe?

Reply:
Made it happen in [`088a77f` (#18293)](https://github.com/gitpod-io/gitpod/pull/18293/commits/088a77f7f1575911f6cc2970c06e168c19c0f1be).